pkg/debounce/debounce.go

```diff
@@ -11,7 +11,8 @@ import (
 // Debouncer struct
 type Debouncer struct {
 	// Used for atomic operations
-	scheduled int32
+	scheduled             *atomic.Bool
+	calledDuringExecution *atomic.Bool
```

Comment:
Why are these pointers? I think these can be regular values. They don't require initialisation that way as well.